docs/utilities.rst

```diff
@@ -23,7 +23,7 @@ You can also override default logging level by setting the environment variable
 
 .. _regress:
 
-Benchmarking and Regression Tests
+Benchmarking and regression tests
```

Comment:
Cool! Could you also move `Tips on the Taichi compiler development`, `ti test`, `ti doc` to here and only left a **link** in contribute.rst? I think they are developer utilities, not contributor general guide.

Reply:
As mentioned above I think those are essential tools for all developers so I left them there.